audiophiler/__init__.py

```diff
@@ -96,7 +96,22 @@ def selected(auth_dict=None):
     return render_template("main.html", db_files=db_files,
                 get_date_modified=get_date_modified, s3_bucket=s3_bucket,
                 auth_dict=auth_dict, harolds=harolds, is_rtp=False,
-                is_eboard=False)
+                is_eboard=False, is_admin=False)
+
+@app.route("/admin")
```

Comment:
I feel like it might be far more intuitive for this to refer to tour mode, since it's just tour mode management, not some all encompassing administrative mode.

Reply:
Again, not my route, back end dictated the routes and their names